### PR TITLE
Update Final Project code

### PR DESCRIPTION
**Pull request by utkarshpant**
# This (draft) PR will

1. Add a `.gitignore` file to make sure only relevant files are tracked for changes
2. Rename the `df` variable to `cancer_data` for readability
3. Remove calls to `setwd` for the time being, to enable code to be run right after cloning
4. Drop the `sample_code_number` column as it is not noteworthy/adds no information for analysis

### CODE REVIEW

**Comment by utkarshpant** [comment]
Will resolve conflicts today, my branch is running a bit behind!

**Comment by SohamAgarkar-ds** [comment]
Hi,

Going shopping for groceries. Will connect with you in the evening!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: FInal Project.R [hunk 1]
```diff
@@ -1,66 +1,94 @@
...
+  geom_histogram(aes(x = clump_thickness),
+                 binwidth = 1,
+                 fill = "#006989",
+                 colour = "#1E1E1E") +
+  labs(x = "Clump thickness",
+       y = "Frequency",
+       title = "Histogram of clump thickness values")
+
+
+# hist(df2$clump_thickness)
```
Comment (utkarshpant) [inline FInal Project.R:76]: Commented for now. Wanted to discuss what diagrams we should be plotting + using `ggplot` to plot graphs.

**Comment by utkarshpant** [comment]
image Added some graphics like this boxplot

**Pull request merged**